packages/embeds/embed-core/src/ModalBox/ModalBox.ts

```diff
@@ -68,17 +83,23 @@ export class ModalBox extends HTMLElement {
       return;
     }
 
-    if (newValue == "loaded") {
-      this.getLoaderElement().style.display = "none";
-    } else if (newValue === "started") {
+    if (newValue === "loading") {
+      this.open();
+      this.hideIframe();
+      this.getLoaderElement().style.display = "block";
+    } else if (newValue == "loaded" || newValue === "reopening") {
```

Comment:
Added `reopening` state that is used when a popup CTA is clicked again, after it has already opened the embed.